events/wc_dark_cult_events.txt

```diff
@@ -1114,9 +1277,52 @@ narrative_event = {
 				character_event = { id = MNM.10030 }
 			}
 			narrative_event = { id = WCDAC.5050 }
+			character_event = { id = WCDAC.5045 days = 90 }
 		}
 	}
 }
+character_event = {
+	id = WCDAC.5045
+	desc = {
+		trigger = {
+			has_global_flag = twilights_hammer_kalimdor
+		}
+		text = EVTDESC_WCDAC_5045_kalimdor
+		picture = GFX_evt_cthun
+	}
+	desc = {
+		trigger = {
+			NOT = { has_global_flag = twilights_hammer_kalimdor }
+		}
+		text = EVTDESC_WCDAC_5045_northrend
+		picture = GFX_evt_yogg_saron
+	}
+	border = GFX_event_normal_frame_intrigue
+
+	is_triggered_only = yes
+	
+	option = {
+		name = EVTOPTA_WCDAC_5045
+		any_courtier = {
+			limit = { culture = twilights_hammer }
+			death = { death_reason = death_sacrificed }
+		}
+		if = {
+			limit = { NOT = { has_global_flag = twilights_hammer_kalimdor } }
+			c_snowdrift = { location = { culture = titanforged } }
+			c_nidavelir = { location = { culture = titanforged } }
+			c_bouldercrag = { location = { culture = titanforged } }
+			grant_title_no_opinion = d_ulduar
+			grant_title_no_opinion = c_ulduar
+		}
+		else = {
+			grant_title_no_opinion = e_ahnqiraj
+			grant_title_no_opinion = c_ahnqiraj
+		}
+		clr_global_flag = twilights_hammer_kalimdor
+		character_event = { id = WCOME.300 }
```

Comment:
What if player controls them?
I'm interested why the Twin Emperors stop ruling the Qiraji.
https://wow.gamepedia.com/Qiraji

Reply:
Cause it makes for better Cho'gall gameplay. Unless you want him going on a world tour and freeing both Cthun and yoggy.

Reply:
What about the Twin Emperors gameplay?

Reply:
Will change to check if they are player and make cho'gall the vassal then

Reply:
The Twin Emperors are rulers of the Qiraji in the lore.
It should be at least otherwise, only if Cho'gall is a player, he gets something.
But to be honest, I would like to see the Twin Emperors there anyway. Cho'gall is a leader of the Cult.

Reply:
Cho'gall didnt even own Silithus in lore